docs/quickstart.md

```diff
@@ -19,29 +19,18 @@ sidebar_label: Quickstart
 
 <div class="emphasize">
     <ul>
-        <li>Kubernetes 1.13+ installed. Latest tested Kubernetes version is 1.17.2.</li>
-        <li>For using features like Local PV and Backup & Restore, you must have Kubernetes version 1.13 or above.
+        <li>Based on the storage requirements for your stateful workloads, <a href="/docs/next/casengines.html#when-to-choose-which-cas-engine"> select the OpenEBS Storage engines to be installed</a>.
```

Comment:
The linked page should include MayaStor as well right?

Reply:
yes. next PR.